### PR TITLE
[EWL-5438] SG2 | Create "Product Nav" Organism

### PR DESCRIPTION
**Pull request by csj1012**
## Ticket(s)

**Jira Ticket**
- [EWL-5438: SG2 | Create "Product Nav" Organism](https://issues.ama-assn.org/browse/EWL-5438)

## Description
Adds the Product Nav organism and adds the new organism to the homepage pattern.

## To Test
- [x] `gulp serve`
- [x] Given that I am viewing SG2
- [x] When I click "Organisms"
- [x] Then I should see "Product Nav"
- [x] When I click "Product Nav"
- [x] Then I should see a horizontal list of white links on a black/dark gray background
- [x] When I click "Pages" in the menu
- [x] Then I should see "Homepage"
- [x] When I click "Homepage"
- [x] Then I should see the "Product Nav"
- [x] And it should be at the top of the homepage
- [x] And it should not be within normal container bounds
- [x] Did you test in IE 11?

## Visual Regressions
https://travis-ci.org/AmericanMedicalAssociation/ama-style-guide-2/builds/403190366?utm_source=github_status&utm_medium=notification

## Relevant Screenshots/GIFs
screen shot 2018-07-11 at 11 24 58 am

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by labbydev** [comment]
I’ve merged in PR #340, please merge in `develop`, remove any unnecessary screenshots, and include links in the VRT section of the PR to the Travis results.

File: styleguide/source/assets/scss/03-organisms/_product-nav.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.ama__product-nav {
+  background: $hoverBlack;
+
+  li {
+    display: inline-block;
+
+    a {
```
Comment (designerneil) [inline styleguide/source/assets/scss/03-organisms/_product-nav.scss:7]: Can we use the class ama__link--white here?

File: styleguide/source/_patterns/03-organisms/product-nav.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="ama__product-nav">
+  <ul>
+    {% for link in productNav.links %}
+      <li>
+        <a href="{{ link.href }}">{{ link.text }}</a>
```
Comment (designerneil) [inline styleguide/source/_patterns/03-organisms/product-nav.twig]: Can we use the link atom here?

**Comment by csj1012** [comment]
@labbydev done, please let me know if I messed anything up the process :P

File: styleguide/source/assets/scss/03-organisms/_product-nav.scss (hunk 1 above)
Comment (csj1012) [reply styleguide/source/assets/scss/03-organisms/_product-nav.scss:7]: I'm now including it via the .json file.

**Review by designerneil — APPROVED** [review]

**Pull request merged**